cmd/ghproxy/ghproxy.go

```diff
@@ -143,6 +145,8 @@ func flagOptions() *options {
 	flag.IntVar(&o.sizeGB, "cache-sizeGB", 0, "Cache size in GB per unique token if using a disk cache.")
 	flag.BoolVar(&o.diskCacheDisableAuthHeaderPartitioning, "legacy-disable-disk-cache-partitions-by-auth-header", true, "Whether to disable partitioning a disk cache by auth header. Disabling this will start a new cache at $cache_dir/$sha256sum_of_authorization_header for each unique authorization header. Bigger setups are advise to manually warm this up from an existing cache. This option will be removed and set to `false` in the future")
 	flag.StringVar(&o.redisAddress, "redis-address", "", "Redis address if using a redis cache e.g. localhost:6379.")
+	flag.StringVar(&o.redisUsername, "redis-usersname", "", "Redis username")
+	flag.StringVar(&o.redisPassword, "redis-password", "", "Redis password")
```

Comment:
I was not able to a full review but I see we add a sensitive value parameter.

1. It should not be passed as a value; a typical pattern in Prow is to pass a path to a file with the secret as content
2. Internally we should work with secrets through the [agent](https://github.com/kubernetes-sigs/prow/tree/main/pkg/config/secret). It handles reading the secret from the file, reloading (maybe not that useful here) and log censoring. Whatever needs the secret will fetch it from the agent.